packages/core/src/connectors/coinbaseWallet.ts

```diff
@@ -135,8 +129,20 @@ export class CoinbaseWalletConnector extends Connector<
         CoinbaseWalletSDK = (<{ default: typeof CoinbaseWalletSDK }>(
           (<unknown>CoinbaseWalletSDK)
         )).default
-
       this.#client = new CoinbaseWalletSDK(this.options)
+
+      // @ts-expect-error – accessing `private` method
+      const walletExtensionChainId = this.#client.walletExtension?.getChainId()
```

Comment:
Will this work with the mobile app? Looks `walletExtension` [only exists in an injected context](https://github.com/coinbase/coinbase-wallet-sdk/blob/78e1fcb95d18fdd6eb4c613738bbe348d10ca59c/packages/wallet-sdk/src/CoinbaseWalletSDK.ts#L234).

Reply:
Yep! Works with the mobile app. It will fallback to the previous behavior if the `walletExtension` is not found.